client-app/pages/company/members.vue

```diff
@@ -3,21 +3,243 @@
     <!-- Title block -->
     <div class="flex justify-between items-center mx-5 md:mx-0">
       <h2 class="text-gray-800 text-3xl font-bold uppercase" v-t="'pages.company.members.title'" />
+      <div v-if="!isMobile" class="space-x-4">
+        <VcButton :is-disabled="true" class="uppercase p-4" is-outline>
+          {{ $t("pages.company.members.buttons.invite_members") }}
```

Comment:
better be chage button text by isMobile condition

Reply:
Container styles are different, I think that this variant is more readable